enaml/widgets/close_event.py

```diff
@@ -0,0 +1,35 @@
+from atom.api import Atom, Bool
```

Comment:
This file needs a copyright header.

Reply:
This is always a bit unclear to me but can the copyright mention 2013 when the file was created only in 2018 ?